ranger/core/filter_stack.py

```diff
@@ -194,6 +194,25 @@ def __str__(self):
         return "<Filter: type == '{ft}'>".format(ft=self.filetype)
 
 
+@stack_filter("vcs_aware")
+class VcsAwareFilter(BaseFilter):
+    vcs_aware_to_function = {
+        VcsAwareFilterConstants.TRACKED:
+        (lambda fobj: fobj.vcsstatus in ['conflict', 'changed', 'changed', 'staged', 'ignored', 'sync']),
```

Comment:
There's a duplicate `'changed'`.

I assume we'd add things like "untracked", "conflict", etc.? Maybe the constant shouldn't be just a letter? And I'm not sure the constants belong in `directory.py` but I don't feel strongly about where they go.